auth/caios-login/action.yml

```
@@ -0,0 +1,36 @@
+name: Log into CAIOS
+description: Authenticate to CAIOS using Github Actions OIDC
+inputs:
+  region:
+    description: "The CoreWeave Region to use, e.g. US-EAST-04A"
+    required: true
+  org_id:
+    description: "The CoreWeave Org ID to use, e.g. cw123456"
+    required: true
+  profile:
+    description: "The AWS CLI profile name to configure. If not set or 'default', uses default profile"
+    required: false
+    default: ""
+
+  # These have defaults
+  audience:
+    description: "The Audience claim to use"
+    required: false
+    default: "https://coreweave.com/iam"
+  endpoint:
+    description: "The OIDC Credentials Endpoint to use"
+    required: false
+    default: "https://api.coreweave.com/v1/cwobject/temporary-credentials/oidc"
+  s3_endpoint:
+    description: "The S3 Endpoint to use. Use http://cwlota.com if using Github runners on CKS"
+    required: false
+    default: "https://cwobject.com"
+
+outputs:
+  access_key_id:
+    description: "The AWS Access Key ID that was created"
+
+runs:
+  using: "node20"
```

https://github.blog/changelog/2025-09-19-deprecation-of-node-20-on-github-actions-runners/
```suggestion
  using: "node24"
```

---

TY!